service/device/registry/internal/src/test/java/org/eclipse/kapua/service/device/registry/event/internal/DeviceEventServiceTest.java

```diff
@@ -35,6 +35,7 @@
 import org.junit.Ignore;
 import org.junit.Test;
 
+@Ignore
```

Comment:
Hi @andrejn-ct ,

I forgot that if we are running Cucumber tests, jUnit tests must be ignored, otherwise they fails.

Regards, 

_Alberto_